papaya-web/src/main/java/pl/edu/agh/papaya/rest/sprints/controller/SprintsController.java

```diff
@@ -0,0 +1,44 @@
+package pl.edu.agh.papaya.rest.sprints.controller;
+
+import com.google.common.collect.Lists;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+import pl.edu.agh.papaya.api.model.SprintIdWithStateDto;
+import pl.edu.agh.papaya.api.model.SprintStateDto;
+import pl.edu.agh.papaya.api.service.SprintsApi;
+import pl.edu.agh.papaya.mappers.SprintStateMapper;
+import pl.edu.agh.papaya.model.Sprint;
+import pl.edu.agh.papaya.model.SprintState;
+import pl.edu.agh.papaya.service.sprint.SprintService;
+
+@RestController
+public class SprintsController implements SprintsApi {
+
+    @Autowired
+    private SprintService sprintService;
+
+    @Autowired
+    private SprintStateMapper sprintStateMapper;
+
+    @Override
+    public ResponseEntity<List<SprintIdWithStateDto>> getSprints(List<SprintStateDto> sprintStateDtos) {
+        List<SprintState> sprintStates = sprintStateMapper.mapFromApi(
+                (sprintStateDtos == null) ? Lists.newArrayList(SprintStateDto.values()) : sprintStateDtos);
```

Comment:
Nie uważam, żeby to był najczytelniejszy ze sposobów zapisu, ale ujdzie

Reply:
Tak na marginesie dałabym mu defaulta z pełną listą, ale boję się drażnić swaggera enumami jeszcze bardziej